### ISSUE TITLE
Adoptium 8u302 installer - IcedTea-Web no longer integrated?

### ISSUE THREAD

**Issue by schindlerd** [adoptium/installer#351]
Is it correct that IcedTea-Web is no longer integrated with the Adoptium installer package?

![adoptium-8u302-no-itw](https://user-images.githubusercontent.com/20475314/136258681-cc9c4923-b6d7-4176-9765-b4e74acc3290.jpg)

The latest AdoptOpenJDK package still had IcedTea-Web integrated:

![adopt-8u292-itw](https://user-images.githubusercontent.com/20475314/136258909-53370f49-9d1b-4a4d-88bf-c91765775ad6.jpg)

**Comment by karianna** [comment]
That's correct, as part of the move to Eclipse Adoptium we, unfortunately, were not able to continue with distributing IcedTeaWeb.  You can still do this yourself via:

https://blog.adoptopenjdk.net/2018/10/using-icedtea-web-browser-plug-in-with-adoptopenjdk/
